docs/source/markdown/podman-auto-update.1.md.in

```diff
@@ -7,39 +7,22 @@ podman\-auto-update - Auto update containers according to their auto-update poli
 **podman auto-update** [*options*]
 
 ## DESCRIPTION
-**podman auto-update** looks up containers with a specified `io.containers.autoupdate` label (i.e., the auto-update policy).
+**podman auto-update** pulls down new container images and restarts containers configured for auto updates.  To make use of auto updates, the container or Kubernetes workloads must run inside a systemd unit.  After a successful update of an image, the containers using the image get updated by restarting the systemd units they run in.  Please refer to `[Quadlet](podman-systemd.unit.5.md)` on how to run Podman under systemd.
 
-If the label is present and set to `registry`, Podman reaches out to the corresponding registry to check if the image has been updated. The label `image` is an alternative to `registry` maintained for backwards compatibility.
-An image is considered updated if the digest in the local storage is different than the one of the remote image.
-If an image must be updated, Podman pulls it down and restarts the systemd unit executing the container.
+To configure a container for auto updates, it must be created with the `io.containers.autoupdate` label or the `AutoUpdate` Quadlet field with one of the following two values:
 
-The registry policy requires a fully-qualified image reference (e.g., quay.io/podman/stable:latest) to be used to create the container.
-This enforcement is necessary to know which image to actually check and pull.
-If an image ID was used, Podman does not know which image to check/pull anymore.
+* `registry`: If the label is present and set to `registry`, Podman reaches out to the corresponding registry to check if the image has been updated. The label `image` is an alternative to `registry` maintained for backwards compatibility.  An image is considered updated if the digest in the local storage is different than the one of the remote image.  If an image must be updated, Podman pulls it down and restarts the systemd unit executing the container.  The registry policy requires a fully-qualified image reference (e.g., quay.io/podman/stable:latest) to be used to create the container.  This enforcement is necessary to know which image to actually check and pull.  If an image ID was used, Podman would not know which image to check/pull anymore.
```

Comment:
I assume a short name alias also works instead of the fully-qualified name?

Reply:
No, only FQNs work.

Short names are accepted by Quadlet (along with a log warning about the risks) but for auto updates I really want to enforce FQNs.

Reply:
Ok seems like an odd choice but I won't argue against it.

Reply:
From a security POV I would prefer Quadlet to not accept short names at all.  But it's just so convenient for quick demos and playing around.

In the case of auto updates, I prefer to force users to be explicit about the sources of images to prevent/protect them from leaking credentials and other short-name issues.  Deploying a Quadlet with short names may yield very different results depending on the node they are being deployed on.  In some cases, it may be intended that different registries are being used (for instance during testing/staging/deployment etc).  In those cases, short names can be (ab)used to pull from different registries but `registries.conf` has better solutions (rewriting of references and mirrors) designed for this use case.

If someone screamed loudly with a compelling use case, we could certainly change the current FQN restriction.